actions/class.RdfController.php

```diff
@@ -1046,19 +1045,29 @@ public function deleteClass()
         );
 
         $class = $this->getClass($id);
+
         if ($this->getRootClass()->equals($class)) {
             $success = false;
+            $deleted = false;
             $msg = __('You cannot delete the root node');
         } else {
             $label = $class->getLabel();
-            $success = $this->getClassService()->deleteClass($class);
-            $msg = $success ? __('%s has been deleted', $label) : __('Unable to delete %s', $label);
+
+            $classDeleter = $this->getPsrContainer()->get(ClassDeleter::class);
+            $classDeleter->delete($class, $this->getRootClass());
+
+            $success = true;
+            $deleted = $classDeleter->isDeleted($class);
+
+            $msg = $success
+                ? __('%s has been deleted', $label)
+                : __('Unable to delete %s', $label);
```

Comment:
$success is always true.

```suggestion
            $msg = __('%s has been deleted', $label);
```

Reply:
I forgot to use `$deleted` instead 😅
Thanks!